libraries/chain/sync_call_context.cpp

```diff
@@ -0,0 +1,21 @@
+#include <eosio/chain/sync_call_context.hpp>
+
+namespace eosio::chain {
+
+sync_call_context::sync_call_context(account_name sender, account_name receiver, uint64_t flags, std::span<const char>data)
+   : sender(sender)
+   , receiver(receiver)
+   , flags(flags)
+   , data(std::move(data))
```

Comment:
I think move of a span might be a bit misleading. span doesn't own the data. Probably better to remove the `std::move`.

Reply:
Thanks. https://github.com/AntelopeIO/spring/pull/1266/commits/bbe2ffab05239a2f3b4228e8dfc88f8c5b0c88e0